code0-identities.gemspec

```diff
@@ -11,7 +11,7 @@ Gem::Specification.new do |spec|
   spec.summary = "Library to manage external identities"
   spec.homepage = "https://github.com/code0-tech/code0-identities"
   spec.license = "MIT"
-  spec.required_ruby_version = ">= 3.0.0"
+  spec.required_ruby_version = ">= 3.1.2"
```

Comment:
We are actually not compatible with `3.0.0` as you can see in this job: https://github.com/code0-tech/code0-identities/actions/runs/11991605452/job/33430499168

> multi_xml-0.7.1 requires ruby version >= 3.1.2, which is incompatible with the current version, 3.0.0